dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -858,6 +871,8 @@ export const decidePalette = (format: ArticleFormat): Palette => {
 			navPillar: borderNavPillar(format),
 			article: borderArticle(format),
 			lines: borderLines(format),
+			secondary: borderSecondary(format),
+			sportBorderTop: sportBorderTop(format),
```

Comment:
Can we change these names. The convention here is to use a name that represents the thing it is colouring.

So, `background.article` is the article background; `border.liveBlock` is the border for the liveBlock, etc.

But with `border.secondary` or `border.sportBorderTop` it isn't clear what they are for. Maybe `border.matchTab` and is the other one about active state? If so, `border.activeMatchTab`